pkg/virtctl/vm/remove_volume_test.go

```diff
@@ -59,7 +63,7 @@ var _ = Describe("Remove volume command", func() {
 			VirtualMachineInstance(metav1.NamespaceDefault).
 			Return(virtClient.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault)).
 			Times(1)
-		virtClient.PrependReactor("put", "virtualmachineinstances/removevolume", func(_ testing.Action) (handled bool, ret runtime.Object, err error) {
+		virtClient.AddReactor("put", "virtualmachineinstances/removevolume", func(_ testing.Action) (handled bool, ret runtime.Object, err error) {
```

Comment:
Because I was trying to figure out why the order in which stuff is executed was not what it was before. And AddReactor appends to the queue where PrependReactor puts it at the top of the queue. But since the mock has no sense of order it doesn't matter. I very much prefer the previous mock since that did have a sense of order. So I had to pass in a function, and in that function set the order in which stuff happens which makes it so much harder to read and understand.